sql/common/src/main/scala/org/apache/sedona/sql/UDF/Catalog.scala

```diff
@@ -148,6 +148,7 @@ object Catalog {
     function[ST_AreaSpheroid](),
     function[ST_LengthSpheroid](),
     function[ST_NumPoints](),
+    function[ST_Force3D](),
```

Comment:
Please use ST_MinimumBoundingCirle's implementation as an example: (1) Function definition: https://github.com/apache/sedona/blob/master/sql/common/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/Functions.scala#LL473C12-L473C36
(2) Function catalog registration: https://github.com/apache/sedona/blob/master/sql/common/src/main/scala/org/apache/sedona/sql/UDF/Catalog.scala#L102

The default value of the optional parameters must be supplied in the Function Catalog otherwise this is not an optional parameter.